test/Table_Tests/src/IO/Excel_Spec.enso

```diff
@@ -73,6 +73,41 @@ spec_fmt suite_builder header file read_method sheet_count=5 =
             t_3.at 'B' . to_vector . should_equal [Nothing, 30]
             t_3.at 'C' . to_vector . should_equal [43.2, 54]
 
+        group_builder.specify "should let you read all sheets from xlsx" <|
+            wb = read_method file
+
+            ## Read all the sheets as single rows.
+            single_table = wb.read_many
+            single_table.row_count . should_equal 25
+            single_table.column_names . should_equal ["SheetName", "Name", "Quantity", "Price", "A", "B", "C", "D", "E", "Student Name", "Enrolment Date", "Item", "Price 1"]
```

Comment:
minor suggestion: Shall it maybe be `Sheet Name` (with a space in between)? Reads a bit nicer IMHO

Reply:
I'm tempted to ask for a test for the edge case where one of the columns in the sheet is named `SheetNames` and so it clashes with the sheet-name-column just to be sure that the de-duplication works correctly.